packages/formatters/CHANGELOG.md

```diff
@@ -0,0 +1,18 @@
+# [@stoplight/spectral-parsers-v1.0.2](https://github.com/stoplightio/spectral/compare/@stoplight/spectral-parsers-v1.0.1...@stoplight/spectral-parsers-v1.0.2) (2022-08-22)
```

Comment:
Wasn't sure what this file should contain, I pulled the initial version from `packages/parsers`.

Reply:
leave it empty for now

Reply:
Done.